docs/docsgen/source/conf.py

```diff
@@ -17,6 +17,18 @@
 release = onnx.__version__
 version = onnx.__version__
 
+# define the latest opset to document,
+# this is meant to avoid documenting opset not released yet
+max_opset = 18
```

Comment:
Why not
```py
max_opset = onnx.helper.VERSION_TABLE[-1][2]
```
That way, it will automatically do the right thing after a new release. Or, do we want to manually control this?

Reply:
That's what the above code will do. The table is supposed to be updated to include 19 only at the time of next release. What am I missing?

Reply:
I'll do that. I was not sure how this table was updated.

Reply:
done